packages/commonwealth/server/routes/communities/put_community_stakes_handler.ts

```diff
@@ -1,17 +1,28 @@
 import { AppError } from '@hicommonwealth/core';
 import {
-  Community,
   CommunityStakeAttributes,
   DB,
   validateCommunityStakeConfig,
 } from '@hicommonwealth/model';
+import { z } from 'zod';
 import { ServerControllers } from '../../routing/router';
 import { TypedRequest, TypedResponse, success } from '../../types';
 import { formatErrorPretty } from '../../util/errorFormat';
 import { validateOwner } from '../../util/validateOwner';
 
-type PutCommunityStakesParams = Community.SetCommunityStakeParams;
-type PutCommunityStakesBody = Community.SetCommunityStakeBody;
+const SetCommunityStakeParamsSchema = z.object({
+  community_id: z.string(),
+  stake_id: z.coerce.number().int(),
+});
+type SetCommunityStakeParams = z.infer<typeof SetCommunityStakeParamsSchema>;
+const SetCommunityStakeBodySchema = z.object({
+  stake_token: z.string().default(''),
+  vote_weight: z.coerce.number().default(1),
```

Comment:
I"m seeing an inconsistency: should this be `stake_scaler` or `vote_weight`?

I thought @kurtisassad updated the property from the former to the latter?

Reply:
Keeeping vote_weight, as defined in model